docs/UserGuide.md

```diff
@@ -18,7 +54,9 @@ App-Ointment is a desktop app for for managing and scheduling patient appointmen
 
 1. Copy the file to the folder you want to use as the _home folder_ for your App-Ointment App.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data. The Ui contains 3 
+   main panels where the doctor list, patient list and appointment list are shown. Updates to the 3 lists will 
+   be shown in their respective panels. The order of the lists follows top to bottom order, i.e. the most recent item will be on the top of the lists <br>
```

Comment:
what does it even mean when the most recent item will be on the top of the list?? this is not correct. 
the last added patient and doctor will appear on the bottom of their respective lists. the appointment with the earlist timeslot (arranged by start time i believe) will appear on the top of the appointment schedule.